File: db/go/sales_management/internal/model/department.go
package model

import "time"

// Department は部門マスタのドメインモデルです
type Department struct {
	DepartmentCode string    `db:"部門コード"`
	StartDate      time.Time `db:"開始日"`
	EndDate        time.Time `db:"終了日"`
	DepartmentName string    `db:"部門名"`
	HierarchyLevel int       `db:"組織階層"`
	DepartmentPath string    `db:"部門パス"`
	IsLowestLevel  int       `db:"最下層区分"`
	CanEnterSlip   int       `db:"伝票入力可否"`
	CreatedAt      time.Time `db:"作成日時"`
	CreatedBy      string    `db:"作成者名"`
	UpdatedAt      time.Time `db:"更新日時"`
	UpdatedBy      string    `db:"更新者名"`
}

File: app/java/src/main/java/com/example/production/domain/model/organization/Department.java
package com.example.production.domain.model.organization;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

/**
 * 部門マスタ
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Department {
    private String departmentCode;
    private String departmentName;
    private LocalDateTime createdAt;
    private String createdBy;
    private LocalDateTime updatedAt;
    private String updatedBy;
}
